### PR TITLE
Update 3WC 2021 to Semifinals

### PR DESCRIPTION
**Pull request by shdewz**

### CODE REVIEW

File: wiki/Tournaments/3WC/2021/en.md [hunk 1]
```diff
@@ -182,6 +223,28 @@ The 3 Digit World Cup is run by various community members.
 
 ## Match results
 
+### Quarterfinals
+
+The stats of the Quarterfinals can be found [here]("https://docs.google.com/spreadsheets/d/146pzh4VK_PuUh2k-AT8T8IRpNWt_da6NmrdTiU0eZR4/edit#gid=1091288063" "Google spreadsheet").
```
Comment (TicClick) [inline wiki/Tournaments/3WC/2021/en.md:228]: please no `here`s, every link has a title, or a generalized meaning you can pass to a reader
```suggestion
The stats of the Quarterfinals can be found [in the spreadsheet]("https://docs.google.com/spreadsheets/d/146pzh4VK_PuUh2k-AT8T8IRpNWt_da6NmrdTiU0eZR4/edit#gid=1091288063" "Google spreadsheet").
```

File: wiki/Tournaments/3WC/2021/en.md [hunk 2]
```diff
@@ -83,28 +83,69 @@ The 3 Digit World Cup is run by various community members.
...
+| Sweden ![][flag_SE] | ![][flag_RU] Russia | **12:30 UTC** | 14:30 UTC+02 | 16:30 UTC+04 | ³ |
+| Sweden ![][flag_SE] | ![][flag_CN] China | **12:30 UTC** | 14:30 UTC+02 | 20:30 UTC+08 | ³ |
+| Taiwan ![][flag_TW] | ![][flag_BR] Brazil | **14:00 UTC** | 22:00 UTC+08 | 11:00 UTC-03 | ¹ |
+| United States ![][flag_US] | ![][flag_XX] Countryless | **15:30 UTC** | 09:30 UTC-06 | 15:30 UTC+00 | ¹ |
+| Germany ![][flag_DE] | ![][flag_FR] France | **17:00 UTC** | 19:00 UTC+02 | 19:00 UTC+02 | ³ |
+| United Kingdom ![][flag_GB] | ![][flag_FR] France | **17:00 UTC** | 18:00 UTC+01 | 19:00 UTC+02 | ³ |
+
+¹ Winner bracket encounter\
+² Loser bracket encounter\
+³ Encounter dependent on results of ² Loser bracket encounter
```
Comment (TicClick) [inline wiki/Tournaments/3WC/2021/en.md:116]: hey, cool stuff

File: wiki/Tournaments/3WC/2021/en.md [hunk 3]
```diff
@@ -83,28 +83,69 @@ The 3 Digit World Cup is run by various community members.
 | ![][flag_GB] | **United Kingdom** | **[Voss](https://osu.ppy.sh/users/7657761)**, [Rinnu](https://osu.ppy.sh/users/12179012), [majoreh](https://osu.ppy.sh/users/7959222), [Kingling](https://osu.ppy.sh/users/7010761), [Jemzsee](https://osu.ppy.sh/users/11553627), [Doomsday](https://osu.ppy.sh/users/18983) |
 | ![][flag_US] | **United States** | **[DigitalHypno](https://osu.ppy.sh/users/4384207)**, [Exarch](https://osu.ppy.sh/users/5370153), [Apraxia](https://osu.ppy.sh/users/4194445), [Woey](https://osu.ppy.sh/users/3792472), [Flezlin](https://osu.ppy.sh/users/3696423), [Tekkito](https://osu.ppy.sh/users/7075211) |
 
-## Match schedule: Quarterfinals
+## Match schedule: Semifinals
 
-### Saturday, 17 April 2021
+This schedule is likely to change as a result of reschedules. Check the [spreadsheet](https://docs.google.com/spreadsheets/d/1oLTe6_WStAfmNU2b6RhyAvZXKkp23dhp5LM6vL6cYuM "Google Spreadsheet") for an up-to-date schedule.
```
Comment (TicClick) [inline wiki/Tournaments/3WC/2021/en.md:88]: too many `schedule`s, but it's ok if you don't find a replacement

**Review by TicClick — DISMISSED** [review]
asd

File: wiki/Tournaments/3WC/2021/en.md [hunk 4]
```diff
@@ -83,28 +83,69 @@ The 3 Digit World Cup is run by various community members.
 | ![][flag_GB] | **United Kingdom** | **[Voss](https://osu.ppy.sh/users/7657761)**, [Rinnu](https://osu.ppy.sh/users/12179012), [majoreh](https://osu.ppy.sh/users/7959222), [Kingling](https://osu.ppy.sh/users/7010761), [Jemzsee](https://osu.ppy.sh/users/11553627), [Doomsday](https://osu.ppy.sh/users/18983) |
 | ![][flag_US] | **United States** | **[DigitalHypno](https://osu.ppy.sh/users/4384207)**, [Exarch](https://osu.ppy.sh/users/5370153), [Apraxia](https://osu.ppy.sh/users/4194445), [Woey](https://osu.ppy.sh/users/3792472), [Flezlin](https://osu.ppy.sh/users/3696423), [Tekkito](https://osu.ppy.sh/users/7075211) |
 
-## Match schedule: Quarterfinals
+## Match schedule: Semifinals
 
-### Saturday, 17 April 2021
+This is likely to change as a result of reschedules. Check the [spreadsheet](https://docs.google.com/spreadsheets/d/1oLTe6_WStAfmNU2b6RhyAvZXKkp23dhp5LM6vL6cYuM "Google Spreadsheet") for up-to-date match times.
```
Comment (TPGPL) [inline wiki/Tournaments/3WC/2021/en.md:88]:
```suggestion
This is likely to change as a result of reschedules. Check the [spreadsheet](https://docs.google.com/spreadsheets/d/1oLTe6_WStAfmNU2b6RhyAvZXKkp23dhp5LM6vL6cYuM "Google spreadsheet") for up-to-date match times.
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**